modules/common/service/service.go

```diff
@@ -176,3 +177,40 @@ func GetServicesListWithLabel(
 
 	return serviceList, nil
 }
+
+// GetServiceWithName - Get service with name in namespace
+func GetServiceWithName(
+	ctx context.Context,
+	h *helper.Helper,
+	name string,
+	namespace string,
+) (*corev1.Service, error) {
+
+	service := &corev1.Service{}
+
+	err := h.GetClient().Get(ctx, types.NamespacedName{Name: name, Namespace: namespace}, service)
+	if err != nil {
+		err = fmt.Errorf("Error getting service %s/%s - %w", name, namespace, err)
+
+		return nil, err
+	}
+
+	return service, nil
+}
+
+// GetServicesPortDetails - Return ServicePort with name from a Service
+func GetServicesPortDetails(
+	ctx context.Context,
+	h *helper.Helper,
```

Comment:
Doesn't look like we need the context or the helper here

Reply:
right, will remove it